### ISSUE TITLE
RFC-API: Not receiving response from login

### ISSUE THREAD

**Issue by gabriel** [keybase/go-libkb#70]
When I do a login and I send the request, I receive requests from keybased for username/password and status messages (identifyUi.warning), which is great, but after it's all completed I'm not getting a response for the original request? (So I don't know that the login succeeded/finished.)

Or I have a bug in my client if this doesn't sound right.

**Comment by maxtaco** [comment]
This  might be a bug in your client because I think my client is working here.  Hmf.

I'm about to close out all of your related issues, give me about 5 minutes

**Comment by gabriel** [comment]
yeah... I guess I'd want RPC methods for key select and key gen, if they
login or signup and don't have a key?

keybase.1.mykey.select(boolean push) returns list of possible keys to
select from

keybase.1.mykey.gen(boolean push) generates a key
Or something like that?

http://glui.me/v1/?i=ftaunf0emqwk28x/2015-01-06_at_5.48_PM.png/

**Comment by maxtaco** [comment]
Got it, these are definitely on  the list of things to do.  I'll bump them up.
